08_project/mohitnag/pkg/puppy/store/store_test.go

```diff
@@ -0,0 +1,128 @@
+package store
+
+import (
+	"fmt"
+	"testing"
+
+	puppy "github.com/anz-bank/go-course/08_project/mohitnag/pkg/puppy"
```

Comment:
no need for the named import if the name is the same as the package name. just use:
`"github.com/anz-bank/go-course/08_project/mohitnag/pkg/puppy"`